<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>学生信息录入</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <layout class="QVBoxLayout" name="MainLayout">
    <item>
     <layout class="QHBoxLayout" name="topPart">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="topLeftLayout">
        <item>
         <layout class="QHBoxLayout" name="nameLayout">
          <item>
           <widget class="QLabel" name="nameLabel">
            <property name="text">
             <string>姓名</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="nameSpacer">
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>30</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="nameEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="idLayout">
          <item>
           <widget class="QLabel" name="idLabel">
            <property name="text">
             <string>学号</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="idSpacer">
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>30</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="idEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="ageLayout">
          <item>
           <widget class="QLabel" name="ageLabel">
            <property name="text">
             <string>年龄</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="ageEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="topSpacer">
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="pictureLayout">
        <item>
         <widget class="QLabel" name="pictureLabel">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pictureButton">
          <property name="text">
           <string>设置图片</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="bottomPart">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="sexLayout">
        <item>
         <widget class="QLabel" name="sexLabel">
          <property name="text">
           <string>性别</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="sexEditMan">
          <property name="text">
           <string>男</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="sexEditWoman">
          <property name="text">
           <string>女</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="academyLayout">
        <item>
         <widget class="QLabel" name="academyLabel">
          <property name="text">
           <string>学院</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="academyEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="courseLayout">
        <item>
         <widget class="QLabel" name="courseLabel">
          <property name="text">
           <string>专业</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="courseEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
